controllers/scripts/create_pod_status_patch.py

```diff
@@ -1,195 +1,267 @@
+#!/usr/bin/env python3
+import os
 import sys
 import json
-import os
-from ipaddress import ip_address, IPv4Address
+import argparse
+import ipaddress
+from pprint import pprint as pp
 
 # Constants
-fileSystemMountPoint = '/workdir/filesystem-volumes'
-blockMountPoint = '/workdir/block-volumes'
-
-
-def executeCommand(command):
-    print('Executing command\n\t' + command)
-    exit = os.system(command)
-    if exit != 0:
-        raise Exception('Error executing command')
+FILE_SYSTEM_MOUNT_POINT = "/workdir/filesystem-volumes"
+BLOCK_MOUNT_POINT = "/workdir/block-volumes"
+BASE_WIPE_VERSION = 6
+ADDRESS_TYPE_NAME = {
+    "access": "accessEndpoints",
+    "alternate-access": "alternateAccessEndpoints",
+    "tls-access": "tlsAccessEndpoints",
+    "tls-alternate-access": "tlsAlternateAccessEndpoints"
+}
 
 
-def getRack(data, podName):
-    print('Checking for rack in rackConfig')
+def process_volumes(pod_name, find, dd, blkdiskard, volumes, effective_method_key):
```

Comment:
Probably wipe_volumes or initialize_volumes is a better name

Reply:
`wipe_volumes ` or `initialize_volumes` names specific cases of the script, IMO the name should be more general than that

Reply:
Let's leave it to be separate functions in case we would like to make wiping to be asynchronous.